config.yaml.default

```diff
@@ -19,6 +19,12 @@ githubToken: your-github-token
 # https://help.github.com/en/github/searching-for-information-on-github/searching-for-repositories
 repoSearch: org:googleapis language:typescript language:javascript is:public archived:false
 
+# By default this toolkit will open a pull request with default branch as the base. If you want to
```

Comment:
I'm struggling to think of a scenario where I'd want to blanket override the default branch.  Generally, I'd expect to use the default branch as the base one when working with the tool.  In the rare event I wanted to orchestrate events against different branches, I could see use for a `--branch` flag on some of the commands, but it's hard to imagine I would want to make this change in a global config.  I wonder out loud if we need to make this configurable at all.

Reply:
I thought about creating a flag, however a configuration option allows the branch override to be associated with the repository as opposed to the command, which made the most sense to me. My main use case for this option would be a repository with a `develop` default branch.

Reply:
I should add in my personal use case always using the default branch would work fine.

Reply:
I think for now leaving this out would be fine.  Limiting this PR to using the default branch instead of master is plenty, and I think we can separate out work to support a *custom* branch.  If that gets tidied up, happy to land this.  And thank you again!